python/paddle/distributed/communication/stream/all_reduce.py

```diff
@@ -79,15 +79,14 @@ def _all_reduce_in_static_mode(
         'all_reduce',
     )
 
-    op_type = _get_reduce_op(op, "allreduce")
```

Comment:
can not delete `op_type = _get_reduce_op(op, "allreduce")` , `op_type` will be used later (L97)

Reply:
yes, i ignore this, my fault.